docs/pages/055_projections.rst

```diff
@@ -1,4 +1,86 @@
 Projections
 ===========
 
-TODO: Add docs here
+The Consistency Manager supports projections for models. This allows using the same ``modelIdentifier`` for multiple different classes. For instance, let's say you have two projections on the same base type - ``PersonModel`` and ``FullProfileModel``. Both models represent the same data (and share fields) so we want to use the same ID for both. However, this doesn't come for free. In general, it's not advised to use projections if you can avoid it and instead use composition (see the Alternatives section of this page).
```

Comment:
"This allows using the same ``modelIdentifier`` for multiple different classes"
It's clearer to say that we're using the same modelIdentifier for _instances_ of different classes.